src/serveses/move-search/MovieSearchService.js

```diff
@@ -1,18 +1,14 @@
-import {filmModels} from "../../models/FilmModel";
 
 export class MovieSearchService {
-  static filterFilms(searchBy, searchValue) {
-    console.log(searchValue)
-    if (searchBy === 'genre') {
-      return filmModels.filter(film => film.genre === searchValue);
+  static sortFilmsByParam(films, param) {
+    if (!films || !param || param.length < 1) {
+      return films;
     }
-    if (searchBy === 'title') {
-      return filmModels.filter(film => film.name === searchValue);
-    }
-    return filmModels;
-  }
-
-  static getFilmById(id) {
-    return filmModels.find(film => film.id.toString() === id.toString());
+    return films.sort((a, b) => {
```

Comment:
method sort mutates an array, so it couldn't been used in reducer, because reducer must do immutable operations otherwise a new data isn't rendered.